sale_order_project/views/project_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="edit_project" model="ir.ui.view">
+        <field name="name">project.project.sale.order.form</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='attachment_tree_view']" position="after">
```

Comment:
here directly use `button`

Ex. `<button name="attachment_tree_view" position="after">`

Reply:
`Element '<button name="attachment_tree_view">' cannot be located in parent view`.